src/CodeGen_C.cpp

```diff
@@ -2429,7 +2440,7 @@ void CodeGen_C::visit(const Call *op) {
             rhs << shape_name;
         } else {
             // Emit a declaration like:
-            // struct {const int f_0, const char f_1, const int f_2} foo = {3, 'c', 4};
+            // struct {int f_0, char f_1, int f_2} foo = {3, 'c', 4};
```

Comment:
This specific example shouldn't actually be possible (at least for Closures), since `pack_struct` code sorts by size.

Reply:
Fixed